components/spicedb/schema/schema.yaml

```diff
@@ -92,7 +92,7 @@ schema: |-
       permission read_billing = member + owner + installation->admin
       permission write_billing = owner + installation->admin
 
-      permission read_prebuild = member + owner + installation->admin
+      permission read_prebuild = collaborator + member + owner + installation->admin
```

Comment:
This org-wide read prebuild permission I flipped on accidentally when looking for the project-specific one, and it ended up raising a question: do we want to let collaborators view the list of prebuilds of the org? I'm guessing not, because we'd have to change those the related pages quite a bit (to signal that you can't start / stop prebuilds or view any of their settings), but it very theoretically could be useful if they wanted to at check what's wrong with a repo's prebuild config and if it's something they can fix themselves.
The detail of a single prebuild still will need some small tweaks to be properly accessible (currently, it just errors), but it probably wouldn't take too long to change.

Reply:
The customer feedback we've gotten seems to indicate we should not let users access the prebuild history.
@geropl I'm going to revert the change and test if it's a safe change to make.